README.md

```diff
@@ -1,3 +1,6 @@
+
+NOTE: This repo has moved to https://github.com/cryptonector/ctp
```

Comment:
github does allow this comment.  Let's import the PR to gerrithub and see what happens.